examples/getting-started-session-based/01-ETL-with-NVTabular.ipynb

```diff
@@ -126,7 +126,7 @@
    "id": "cd861fcd",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    # Group interaction features by session and sorted by timestamp
</pre><p>I think it should be Group interaction features by session` . Looks like we do not sort by timestamp but we have to. We are missing that step.. May be you can add this step as in this ETL notebook https://github.com/NVIDIA-Merlin/Transformers4Rec/blob/main/examples/end-to-end-session-based/01-ETL-with-NVTabular.ipynb  in cell 10 ..   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/548/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
There is no timestamp column in the synthetic dataset.